guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -323,17 +286,26 @@ php bin/magento setup:upgrade
 php bin/magento maintenance:disable
 ```
 
-### (Optional) Restart Varnish
+### Restart Varnish
 
-Restart Varnish if you use it for page caching.
+_Optional_—If you use Varnish for page caching, restart it:
 
 ```bash
 service varnish restart
 ```
 
-### Check your storefront online
+### Check your work
+
+Open your storefront URL in a web browser to check whether the upgrade was successful. If your upgrade was unsuccessful, your storefront will not load properly.
 
-Open your storefront to check if the upgrade was successful.
+If the application fails with a  `We're sorry, an error has occurred while generating this email.` error:
+
+1. Reset [file system ownership and permissions] as a user with `root` privileges.
+1. Clear these directories:
+   * `<your Magento install dir>/var/cache`
+   * `<your Magento install dir>/var/page_cache`
+   * `<your Magento install dir>/generated/code`
+1. Check our storefront in your web browser again.
```

Comment:
it is not clear what means "our storefront"

Reply:
That was a mistake! It should be "your storefront". Changing.